### PR TITLE
Add info about updating contacts using subscription API

### PR DESCRIPTION
**Pull request by admiral96**
Cards Related: [#4634](https://github.com/engagespark/engageSPARK-frontend/issues/4634)

### CODE REVIEW

**Comment by mknecht** [comment]
@avnermiz should we name those “custom fields” in public documentation? What is the official name we should use, going forward, in the documentation, and the API itself?

**Comment by avnermiz** [comment]
let's use custom fields - it seems to be the term that people use across the board, especially devs. @mknecht @admiral96 

also, let's not wait for me on these kinds of decisions, as I may not always have time to respond.  For naming decisions, please either (a) email me with an urgent if you think i need to weigh in or (b) just make a decision.  It seems this got delayed by 19 days waiting for me to decide on a naming issue; that's really not good.

**Comment by mknecht** [comment]
@avnermiz agreed. I think it's mostly because it's a non-code PR, so it's more like it got forgotten. :/

@admiral96 please remember to check *all* your PRs with the GH searach and push them forward

**Comment by bnx05** [comment]
will check in a bit

File: docs/index.md [hunk 1]
```diff
@@ -232,20 +230,81 @@ Example for a subscription API URL which enable contact re-use:
 
     https://start.engagespark.com/api/v1/engagements/1234/contacts/?try_using_existing=newest
 
-**Current catch**: All other parameters of the contacts, whose existing pendant is found, are discarded.
```
Comment (bnx05) [inline docs/index.md]: i removed this note since you can update an existing contact when using the `try_using_existing` parameter

Comment (bnx05) [reply docs/index.md]: @admiral96 please confirm, i am not sure i understand what that sentence meant.

Comment (admiral96) [reply docs/index.md]: :+1: I don't understand it either but it's good that you removed it(I forgot why). Thanks

**Comment by mknecht** [comment]
@admiral96 get this merged today or close, thank you.

**Pull request merged**